packages/manager/modules/exchange/package.json

```diff
@@ -12,10 +12,21 @@
   "main": "src/index.js",
   "dependencies": {
     "filesize": "^3.6.1",
-    "lodash": "^4.17.11"
+    "lodash": "^4.17.11",
+    "moment": "^2.24.0"
   },
   "peerDependencies": {
+    "@ovh-ux/ng-ovh-export-csv": "^1.0.0",
```

Comment:
Missing both:

`"@ovh-ux/ng-ovh-otrs": "^7.1.7",`

and

`"angular-ui-bootstrap": "1.3.3",`